addons/backup-and-maintenance.sh

```diff
@@ -84,6 +84,13 @@ should_backup(){
     # Default choices
     SHOULD_BACKUP=1
     MARIADB_LOCAL_CLUSTER=0
+
+    if [ $MARIADB_REMOTE_CLUSTER -eq 1 ]; then
+        echo "Server not configured to do DB backups: database backup canceled."
```

Comment:
canceled -> cancelled

Reply:
I would probably just write to be consistent:

"Server is not configured to do database backups: the operation has been cancelled."

Reply:
I agree with you @extrafu. I prefer: "Remote database detected: backup should be done on database server itself."

Reply:
Fine to me!